components/org.wso2.carbon.identity.api.server.application.management/org.wso2.carbon.identity.api.server.application.management.v1/src/main/resources/applications.yaml

```diff
@@ -3061,6 +3061,10 @@ components:
           description: Decides whether the application is visible for end users.
         certificate:
           $ref: '#/components/schemas/Certificate'
+        applicationAccessEnabled:
+          type: boolean
+          example: true
+          description: Decides whether the application can be accessed.
```

Comment:
Better to treat this property as a primary property as similar to  isManagementApp 
 https://github.com/wso2/identity-api-server/pull/612/files#diff-37b5c0ba2971cd297cefdc03e49736637886f855740d6071634220bdf1f15615R2921

Reply:
Addressed with https://github.com/wso2/identity-api-server/pull/612/commits/72322e078e04a1e5a6bb598c202f047c4a807070